docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,299 @@
+---
+title: "Zaaktypen-API"
+date: '19-01-2019'
+---
+
+# ZAAKTYPEN API
+
+FUNCTIONELE BESCHRIJVING
+
+Hieronder lichten we de gebruiksmogelijkheden van deze API toe. Voor de
+werking van deze API verwijzen wij u naar de OAS-specificatie (zie
+‘link’ hieronder).
+
+<table>
+<tbody>
+<tr class="odd">
+<td><strong>Aspect</strong></td>
+<td><strong>Beschrijving</strong></td>
+</tr>
+<tr class="even">
+<td>Doel</td>
+<td>Het kunnen aanspreken van een voorziening voor het onderhouden en raadplegen van zaken inclusief bijbehorende statussen, rollen en relaties naar betrokkenen, objecten, zaaktypen en statustypen.</td>
```

Comment:
Dit is een copy-paste van zaken-api documentatie? Moet m.i. focussen op zaaktypen/informatieobjecttypen/resultaattypen/objecttypen etc.

Reply:
Oeps.. Omschrijving is aangepast.